README.md

```diff
@@ -8,6 +8,10 @@
 
 ![Logo](https://hacktoberfest.com/_next/static/media/opengraph.da6e44c0.png)
 
+
+#Please Add Excluded ones Here:
```

Comment:
Hmmmm, I think that may be a good idea mapping excluded repositories, but not with this title, and not in the beginning. I would accept a new file called "EXCLUDED_2022.md" and a mention to it in the end of the list